### PR TITLE
Logan - More components

### PR DESCRIPTION
**Pull request by loganwright**
Not sure if the view extension stuff is the best implementation, but I find myself rewriting `CGRectGetWidth` a million times per project, and I think it'll be helpful.

### CODE REVIEW

File: src/Async/After/After.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+public func After(after: NSTimeInterval, op: () -> (), completion: (() -> Void)?) {
+    let seconds = Int64(after * Double(NSEC_PER_SEC))
+    let dispatchTime = dispatch_time(DISPATCH_TIME_NOW, seconds)
+    dispatch_after(dispatchTime, dispatch_get_main_queue()) {
+        let blockOp = NSBlockOperation(block: op)
+        blockOp.completionBlock = completion
+        NSOperationQueue.mainQueue().addOperation(blockOp)
+    }
+}
```
Comment (paulrolfe) [inline src/Async/After/After.swift]: new line at end of file if you wanna, not that important.

File: src/Geometry/Geometry.swift [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    public var radians: CGFloat {
+        return CGFloat(Double(self) * (M_PI / 180))
+    }
+}
+
+public extension CGFloat {
+    public var degrees: CGFloat {
+        return self * CGFloat(180.0 / M_PI)
+    }
+}
```
Comment (paulrolfe) [inline src/Geometry/Geometry.swift:21]: Curious why you don't combine this with the `CGFloat` extension above it? I see this a lot.

Comment (loganwright) [inline src/Geometry/Geometry.swift:21]: Because I made them at different times, and threw them all in this file :)  Will fix.

**Comment by paulrolfe** [comment]
:goat: :seedling:

File: src/Async/After/After.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        wrappedCompletion = completion
+    }
+    
+    After(after, op: op, completion: wrappedCompletion)
+}
+
+public func After(after: NSTimeInterval, op: () -> (), completion: (() -> Void)?) {
+    let seconds = Int64(after * Double(NSEC_PER_SEC))
+    let dispatchTime = dispatch_time(DISPATCH_TIME_NOW, seconds)
+    dispatch_after(dispatchTime, dispatch_get_main_queue()) {
```
Comment (brightredchilli) [inline src/Async/After/After.swift]: I feel we should guard that we are on the main thread when this method gets called. or we can rename the block to completion on main thread. this behaviour is sort of undocumented in this library.

File: src/UIKit/UIView+Geometry.swift [hunk 4]
```diff
@@ -0,0 +1,69 @@
+//
+//  UIView+Geometry.swift
+//
+//  Created by Logan Wright on 10/24/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+public extension UIView {
```
Comment (brightredchilli) [inline src/UIKit/UIView+Geometry.swift]: i feel like these methods play a lot nicer as cgrect extensions than uiview extensions.

then 
view.frame.width
view.bounds.halfwidth

just works without us having to define separate methods. and i think it adds some readability.

File: src/Async/After/After.swift (hunk 3 above)
Comment (loganwright) [inline src/Async/After/After.swift]: Good point, do you think it should run on the calling thread, or document main, or provide an option to specify and maybe default to main queue

File: src/UIKit/UIView+Geometry.swift (hunk 4 above)
Comment (loganwright) [inline src/UIKit/UIView+Geometry.swift]: That was exactly my concern!  I'll swap them out

**Pull request merged**